books/std/lists/prefixp.lisp

```diff
@@ -164,4 +166,26 @@ the list @('y')."
                   (equal (take n l1) (take n l2)))
              (equal (prefixp (nthcdr n l1) (nthcdr n l2))
                     (prefixp l1 l2)))
-    :hints (("goal" :in-theory (enable prefixp)))))
+    :hints (("goal" :in-theory (enable prefixp))))
+
+  (defthm prefixp-one-way-or-another
```

Comment:
Seems good.  Going to let @acoglio review it before merging.

Reply:
Looks fine to me, thanks.